docs/getting-started/quickstart.md

```diff
@@ -11,45 +11,56 @@ search:
 
 # Quickstart
 
-Prefect is an orchestration and observability platform that empowers developers to build and scale code quickly, turning their Python scripts into resilient, recurring workflows.
-
-In this quickstart, you'll see how you can schedule your code on remote infrastructure and observe the state of your workflows.
-With Prefect, you can go from a Python script to a production-ready workflow that runs remotely in a few minutes.
-
-Let's get started!
+Prefect is a workflow orchestration platform for building data pipelines, background jobs, LLM Chains, and more. With Prefect, you can elevate a Python script into a scheduled, observable, resilient, remotely execuateble workflow in a few minutes. Let's get started!
 
 ## Setup
 
 Here's a basic script that fetches statistics about the [main Prefect GitHub repository](https://github.com/PrefectHQ/prefect).
 
-```python
-import httpx
+```python title="my_gh_workflow.py"
+import httpx   # an HTTP client library
 
-def get_repo_info():
-    url = "https://api.github.com/repos/PrefectHQ/prefect"
-    response = httpx.get(url)
-    repo = response.json()
-    print("PrefectHQ/prefect repository statistics 🤓:")
-    print(f"Stars 🌠 : {repo['stargazers_count']}")
+def get_repo_info(repo_owner: str, repo_name: str):
+    """Get info about a repo"""
+    url = f"https://api.github.com/repos/{repo_owner}/{repo_name}"
+    api_response = httpx.get(url)
+    api_response.raise_for_status()
+    repo_info = api_response.json()
+    return repo_info
+
+def get_contributors(repo_info: dict):
+    """Get contributors for a repo"""
+    contributors_url = repo_info["contributors_url"]
+    response = httpx.get(contributors_url)
+    response.raise_for_status()
+    contributors = response.json()
+    return contributors
+
+def repo_info(repo_owner: str = "PrefectHQ", repo_name: str = "prefect"):
+    """Given a repo, logs the number of stargazers and contributors"""
+    repo_info = get_repo_info(repo_owner, repo_name)
+    print(f"Stars 🌠 : {repo_info['stargazers_count']}")
+
+    contributors = get_contributors(repo_info)
+    print(f"Number of contributors 👷: {len(contributors)}")
 
 if __name__ == "__main__":
-    get_repo_info()
+    repo_info()
 ```
 
-Let's make this script schedulable, observable, resilient, and capable of running anywhere.
+Copy this script into a file named `my_gh_workflow.py`. Run it locally to ensure your Python environment works. Let's make this script schedulable, observable, resilient, and capable of running anywhere.
```

Comment:
Is the "let's make this" section still meant to be in here? It feels a little disjointed.

Reply:
I think it's meant to be a transition from "you just have a script" to "look what Prefect can do for you".  Is that transition improved by the suggested change added below?